### PR TITLE
feat: add script to update nodes

### PR DESCRIPTION
**Pull request by josepfo**

### CODE REVIEW

**Comment by josepfo** [comment]
[Update wallet nodes upon update of lamassu-server](https://trello.com/c/3VeVACyF/936-update-wallet-nodes-upon-update-of-lamassu-server)

File: bin/lamassu-update-wallet-nodes [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env node
+
+const _ = require('lodash/fp')
+
+const { utils: coinUtils } = require('lamassu-coins')
+const cryptos = coinUtils.cryptoCurrencies()
+
+const common = require('../lib/blockchain/common')
```
Comment (chaotixkilla) [inline bin/lamassu-update-wallet-nodes]: Group this `require(...)` with the other ones

File: bin/lamassu-update-wallet-nodes [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+const PLUGINS = {
+  BTC: require('../lib/blockchain/bitcoin.js'),
+  LTC: require('../lib/blockchain/litecoin.js'),
+  ETH: require('../lib/blockchain/ethereum.js'),
+  DASH: require('../lib/blockchain/dash.js'),
+  ZEC: require('../lib/blockchain/zcash.js'),
+  BCH: require('../lib/blockchain/bitcoincash.js')
+}
```
Comment (chaotixkilla) [inline bin/lamassu-update-wallet-nodes:16]: While this works for current coins, the script will need manual maintenance for every new coin added to the system. Is something like this possible?

```suggestion
const PLUGINS = {}
_.forEach(it => PLUGINS[it.cryptoCode] = require(`../lib/blockchain/${it.code}.js`), cryptos)
```

Comment (chaotixkilla) [reply bin/lamassu-update-wallet-nodes:16]: I guess that eventually manual maintenance will be needed, as the coin script will need to be developed, so you can ignore this I think

**Review by chaotixkilla — CHANGES_REQUESTED** [review]

File: bin/lamassu-update-wallet-nodes (hunk 2 above)
Comment (josepfo) [reply bin/lamassu-update-wallet-nodes:16]: Yeah, I think we can't run from it other than placing this in a common file

**Review by chaotixkilla — APPROVED** [review]

**Pull request merged**